airflow/operators/bash_operator.py

```diff
@@ -38,9 +38,6 @@ class BashOperator(BaseOperator):
     :param bash_command: The command, set of commands or reference to a
         bash script (must be '.sh') to be executed. (templated)
     :type bash_command: str
-    :param xcom_push: If xcom_push is True, the last line written to stdout
-        will also be pushed to an XCom when the bash command completes.
```

Comment:
I think  the info here that just the last line is returned to XCom needs to be preserved somewhere? (Possibly just as a new paragraph on line 37, before the params)